source/_components/switch.pilight.markdown

```diff
@@ -49,13 +49,15 @@ Variables for the different codes (`on_code` and `off_code`):
 
 - **protocol** (*Required*): Protocol to use, eg. `intertechno_old` or `daycom`.
 - **systemcode** (*Optional*): The systemcode of the device.
-- **unit** (*Optional*): The unit to use.
+- **unit** (*Optional*): The unit to use (is equivalent to pilight-send --unit).
+- **unitcode** (*Optional*): The unitcode to use (is equivalent to pilight-send --unitcode).
+- **programcode** (*Optional*): The programcode to use (is equivalent to pilight-send --programcode).
```

Comment:
I guess this must be `systemcode`?

https://github.com/home-assistant/home-assistant/blob/af5d0b3443d9f531f40ee8d7ffd52ad548e7e9d6/homeassistant/components/switch/pilight.py#L25

At least, there is no `programcode` in the source.

Reply:
there is a programcode in their documentation: 
e.g. as a parameter for pilight-send: https://manual.pilight.org/programs/send.html?highlight=programcode
and as a parameter for the impuls-protocol: https://manual.pilight.org/protocols/433.92/switch/impuls.html?highlight=programcode

Reply:
Well yeah, but this is not the pilight documentation, it is Home Assistant.
I cannot find the option you are adding to the documentation, so I cannot accept/merge this PR.

Could you please update this PR with the correct parameters that match the Home Assistant code base?